crd-docs/cr/kiali.io_v1alpha1_kiali.yaml

```diff
@@ -371,6 +371,7 @@ spec:
       enabled: true
     disabled_features: []
     istio_injection_action: true
+    istio_annotation_action: true
```

Comment:
don't forget to sort these in alphabetical order. So "istio_annotation_action" should come before "istio_injection_action".. do that in all the files where this new setting is added.

Reply:
Omg sorry I forgot it working on it !! I'll change it thanks @jmazzitelli